injectors/index.js

```diff
@@ -48,13 +48,38 @@ if (!VALID_PLATFORMS.includes(platform)) {
   }
 
   if (process.argv[2] === 'inject') {
-	try {
-		result = await main.inject(platformModule, platform);
-	} catch (e) {
-		// this runs if path generator crashes (app folder doesnt exist)
-		console.log(`${AnsiEscapes.RED}Platform you specified isn't installed on this device!${AnsiEscapes.RESET}\n\nList of valid platforms:\n${AnsiEscapes.GREEN}${VALID_PLATFORMS.map(x => `${x}`).join('\n')}${AnsiEscapes.RESET}`);
-		process.exit(process.argv.includes('--no-exit-codes') ? 0 : 1);
-	}
+    try {
+      if (!process.argv.find(x => VALID_PLATFORMS.includes(x.toLowerCase()))) {
```

Comment:
```suggestion
      if (!VALID_PLATFORMS.includes(process.argv[3]?.toLowerCase())) {
```
Should've been done in #172

Reply:
This is not handled yet

Reply:
I made a shared function to handle this already on line 48 for both un/injecting.

Reply:
https://github.com/replugged-org/replugged/blob/bd20ab37ab91508278aceb409694b1446f0301e7/injectors/index.js#L31

Reply:
Should be handled now

Reply:
Still not

Reply:
Don't understand what still needs fixing here?

Reply:
You're checking all the args. Only one should be checked.

Reply:
Fixed